ui/packages/catalog-ui-search/src/main/webapp/component/filter-comparator/filter-comparator.view.js

```diff
@@ -20,11 +20,13 @@ const template = require('./filter-comparator.hbs')
 const CustomElements = require('../../js/CustomElements.js')
 const metacardDefinitions = require('../singletons/metacard-definitions.js')
 
-var geometryComparators = ['INTERSECTS']
-var dateComparators = ['BEFORE', 'AFTER', 'RELATIVE', 'BETWEEN']
-var stringComparators = ['CONTAINS', 'MATCHCASE', '=', 'NEAR']
-var numberComparators = ['>', '<', '=', '>=', '<=']
-var booleanComparators = ['=']
+var geometryComparators = ['INTERSECTS', 'EMPTY']
+var geometryComparatorsAnyGeo = ['INTERSECTS']
```

Comment:
That specific search is not part of the ticket. Additionally, I'm not sure if it would be easy to gather all the geometric attributes of a metacard and perform a search on all such attributes.